src/particles/ImpactXParticleContainer.cpp

```diff
@@ -79,9 +83,6 @@ namespace impactx
         // write Real attributes (SoA) to particle initialized zero
         DefineAndReturnParticleTile(0, 0, 0);
 
-        pinned_tile.push_back_real(RealSoA::ux, np, 0.0);
-        pinned_tile.push_back_real(RealSoA::uy, np, 0.0);
-        pinned_tile.push_back_real(RealSoA::pt, np, 0.0);
         pinned_tile.push_back_real(RealSoA::ux, *px.cbegin(), *px.cend());
         pinned_tile.push_back_real(RealSoA::uy, *py.cbegin(), *py.cend());
         pinned_tile.push_back_real(RealSoA::pt, *pz.cbegin(), *pz.cend());
```

Comment:
This did hit the wrong overload:
```suggestion
        pinned_tile.push_back_real(RealSoA::ux, &(*px.cbegin()), &(*px.cend()));
        pinned_tile.push_back_real(RealSoA::uy, &(*py.cbegin()), &(*py.cend()));
        pinned_tile.push_back_real(RealSoA::pt, &(*pz.cbegin()), &(*pz.cend()));
```

Reply:
Will simplify later on to
```suggestion
        pinned_tile.push_back_real(RealSoA::ux, px);
        pinned_tile.push_back_real(RealSoA::uy, py);
        pinned_tile.push_back_real(RealSoA::pt, pz);
```
when https://github.com/AMReX-Codes/amrex/pull/2634 is merged.

Reply:
Simplification in #79